src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -261,6 +261,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       for {
         _ <- backend.cleanUpForWorkflow(workflow)
         _ <- globalDataAccess.updateWorkflowOptions(workflow.id, workflow.workflowOptions.clearEncryptedValues)
+        _ <- if(toState == WorkflowSucceeded) workflow.postProcessWorkflow else Future.successful({})
```

Comment:
Absolutely, I needed a way to see what happened when postProcessWorkflow was called with a real workflow